MathGame.Chad1082/MathGame.Chad1082/Helpers.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MathGame.chad1082
+{
+    internal class Helpers
+    {
+        private static List<string> gameList = new List<string>();
```

Comment:
We can shorten this as follows:

```suggestion
        private static List<string> gameList = new();
```